doc/DEVELOPMENT.rst

```diff
@@ -249,6 +249,6 @@ modified:
 
 ::
 
-    scripts/gen-atlas.py atlas/overrides atlas resources/00-taisei.pkgdir/gfx
+    ninja gen-atlas-common_ui
```

Comment:
Ah well, I see you edited this before I submitted my earlier comment. It's still not quite right though, so that comment still applies - this command will regenerate just the `common_ui` atlas specifically.